### PR TITLE
Feat added homescreen

### PR DESCRIPTION
**Pull request by ianuragsharma**
Please review the last two commit (feat added Homescreen and initial setup).

image

### CODE REVIEW

**Review by hrishikeshmane — APPROVED** [review]
Looks alright. Good to merge.

File: src/components/category/category.css [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  position: absolute;
+  left: 50%;
+  top: 50%;
+  transform: translate(-50%, -50%);
+  width: 100%;
+  font-weight: 900;
+  background-color: var(--dim-grey);
+}
+
+@media screen and (max-width: 500px) {
```
Comment (ishitakeshawani) [inline src/components/category/category.css]: You can change the width to em.

File: src/screens/HomeScreen/homescreen.css [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  border-top: 2px solid var(--dim-grey);
+  margin: auto 10px;
+}
+.product-list {
+  margin-top: 1rem;
+  justify-content: space-around;
+  flex-wrap: wrap;
+}
+
+@media screen and (max-width: 500px) {
```
Comment (ishitakeshawani) [inline src/screens/HomeScreen/homescreen.css]: Use em here.

**Review by ishitakeshawani — APPROVED** [review]
Added a suggestion. 
I liked that you have used barrel export.

File: src/screens/HomeScreen/homescreen.css (hunk 2 above)
Comment (ianuragsharma) [reply src/screens/HomeScreen/homescreen.css]: ok Thanks for pointing out

**Pull request merged**